src/sumo_store_mysql.erl

```diff
@@ -483,3 +484,28 @@ hash(Clause) ->
   List = binary_to_list(Bin),
   Fun = fun(Num) -> string:right(integer_to_list(Num, 16), 2, $0) end,
   lists:flatmap(Fun, List).
+
+%% @private
+wakeup(Doc) ->
+  Fields = sumo_store_utils:fields_from_doc(Doc),
+  lists:foldl(fun({FieldName, FieldType, FieldValue}, Acc) ->
```

Comment:
I think that writing code that is specific to each store is the best approach, so let's do that!!!